web_m2x_options/models/model.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields
+
+
+class IrModel(models.Model):
+    _inherit = 'ir.model'
+
+    disable_quick_create = fields.Boolean('Disable quick create')
```

Comment:
Would you mind naming this field `avoid_quick_create` so users of `base_optional_quick_create` can easily migrate to your approach (which is more elegant)